src/label/labelStyle.ts

```diff
@@ -421,6 +421,11 @@ function setTextStyleCommon(
     if (margin != null) {
         textStyle.margin = margin;
     }
+    const ellipsis = textStyleModel.get('ellipsis');
+    if (ellipsis) {
+        textStyle.ellipsis = ellipsis;
+    }
+    textStyle.lineOverflow = textStyleModel.get('lineOverflow') || 'truncate';
```

Comment:
In this way, developers still can't disable it by setting the option to null. We should avoid applying default value in the code. Instead, default value is better to be managed in the `defaultOption`

Reply:
Done